test/util.py

```diff
@@ -125,7 +119,7 @@ def umount(mount_process, mnt_dir):
         if code is not None:
             if code == 0:
                 return
-            pytest.fail('file system process terminated with code %s' % (code,))
+            pytest.fail('file system process terminated with code {}'.format(code))
```

Comment:
not needed.

i guess you did not change **all** percent to `.format()`, did you?

so changing these 2 is not needed either.

Reply:
Nope, not all. I used the `pyupgrade` tool which doesn't replace them all. I've removed them.

Reply:
thanks!